app/util/conf/base_config_loader.py

```diff
@@ -135,6 +138,20 @@ def load_from_config_file(self, config, config_filename):
         if self.CONFIG_FILE_SECTION:
             self._load_section_from_config_file(config, config_filename, self.CONFIG_FILE_SECTION)
 
+    def _override_cacerts(self):
+        """
+        Override path for requests certs if this is a frozen executable run.
+        SSL cacerts cannot be found when running frozen CR package. Workaround the issue by bundling the cacert.pem
+        file into frozen package and at startup update the environment to point to these cacerts.
+        """
+        if getattr(sys, 'frozen', False):
+            root_directory = dirname(sys.executable)
+            cacert_file = '{}/cacert.pem'.format(root_directory)
+            if isfile(cacert_file):
+                environ['REQUESTS_CA_BUNDLE'] = cacert_file
+            else:
+                raise FileNotFoundError(errno.ENOENT, strerror(errno.ENOENT), cacert_file)
+
```

Comment:
This can be done with `os.stat(filename)` which will throw FileNotFound exception. But I think raising an exception manually is cleaner approach.

Reply:
What do you think about whether to catch this exception in main and print and exit(1) or let the stack trace get printed and continue?

So in main:_initialize_configuration
```
try:
        conf_loader.configure_postload(config)
except FileNotFoundError:
        print "Error etc."
        sys.exit(1)
```

Reply:
Eh, I think it's fine to print the stack trace. It's not really a user error.